doc/CompilationPython.md

```diff
@@ -0,0 +1,189 @@
+| [Tutorials Home](index.md) | [Previous](ImportExport.md) | [Next](PythonModule.md) |
+| :--- | :---: | ---: |
+
+# Compiling libpointmatcher with python
+
+This tutorial presents the different steps of compiling pypointmatcher, the libpointmatcher's python module, on Ubuntu and Mac OS X.
+
+## Prerequisites
+
+To get started, you will obviously need the same prerequisites as libpointmatcher, but also some additional dependencies as listed here :
+
+| Name | Version <br> (Tested August 2020 on Ubuntu 18.04) |
+| :--- | :---: |
+| pybind11 | 2.5.0 |
+| Python3 | 3.6.9 |
+|||
+| **Dependencies** | |
+| catch | 1.10.0 |
+| python3-dev | 3.6.7 |
+| pytest | 5.4.3 |
+
+`catch` and `python3-dev` need to be installed with the package manager :
+
+```bash
+sudo apt install catch python3-dev
+```
+
+and `pytest` with `pip` :
+
+```bash
+pip3 install pytest
+```
```

Comment:
The above instructions are only for Ubuntu. Could you specify that?

Reply:
For the apt package manager yes, I could add the macOS command. But for the pip command, isn't it the same for macOS and Ubuntu?

Reply:
the `python3-dev` package doesn't have to be installed on macOS, the [documention](https://pybind11.readthedocs.io/en/latest/basics.html#linux-macos) says that the included python version works out of the box. I can specify that instead.